src/LandPlanner.cpp

```diff
@@ -72,8 +83,11 @@ bool LandPlanner::prepareForTakeover(const ros::Time& time)
         return false;
     }
 
-    requested_height_ = transform.transform.translation.z;
+    requested_height_ = transform.transform.translation.z; 
 
+    // height determined by the ratio of landing accelerations
+    cushion_height_ = std::min(0.5 * (std::pow(descend_rate_,2)/cushion_acceleration_), 
```

Comment:
What's the reasoning behind these two?  You don't need to go through all the math here, but there should be a high-level description somewhere of what assumptions you're starting with